tests/scr-readingsystem-features/OPS/content_001.xhtml

```diff
@@ -0,0 +1,88 @@
+        'keyboard-events',
+        'spine-scripting'
+       ]
+
+      /**
+       * A simple change that proves that scripting is enabled.
+       * 
+       * Note that the script is more complicated than would be necessary in HTML, because the innerHTML method does not
+       * seem to work in xhtml (only in html) :-(
+       */ 
+      function test_ers_features() {
+        const support = document.getElementById('scripting_support');
+        support.textContent = 'This Reading System supports scripting.';
+
+        try {
+          const ers = navigator.epubReadingSystem;
+          // For code test only!!
+          // const ers = {
+          //   hasFeature(n) {
+          //     if (required_features.includes(n)) {
+          //       if (n === 'keyboard-events') return false;
+          //       if (n === 'touch-events') return undefined;
+          //       return true;
+          //     } else {
+          //       return false;
+          //     }
+          //   }
+          // }
+          if (ers !== undefined) {
+            const div_pass = document.getElementById('pass');
+            const div_fail = document.getElementById('fail');
+
+            for (const feature of required_features) {
+              const feature_entry = document.getElementById(feature);
+              try {
+                const result = ers.hasFeature(feature);
+                feature_entry.textContent = (result === undefined || result === null) ? 'n/a' : `${result}`;
+              } catch(e) {
+                // if whatever happens...
+                feature_entry.textContent = 'n/a';
+              }
+            }
+
+            div_fail.style.display = 'none';
+            div_pass.style.display = 'block';
+          }
+        } catch (e) {
+          // Let the original, i.e., negative report, stay as is. 
+        }
+      }
+
+      window.addEventListener('load', test_ers_features);
+    </script>
+  </head>
+  <body>
+    <p id="scripting_support">This Reading System does not support scripting.</p>
+
+    <div id="fail" style="display: block">
+      <p>The Reading System does not implement the <code>epubReadingSystem</code> object. Test fails.</p>
+    </div>
+
+    <div id="pass" style="display: none">
+      <p>The Reading System implements the <code>epubReadingSystem</code> object with the following features:</p>
+      <ul>
+        <li><code>dom-manipulation</code>: <code id="dom-manipulation"></code></li>
+        <li><code>layout-changes</code>: <code id="layout-changes"></code></li>
+        <li><code>touch-events</code>: <code id="touch-events"></code></li>
+        <li><code>mouse-events</code>: <code id="mouse-events"></code></li>
+        <li><code>keyboard-events</code>: <code id="keyboard-events"></code></li>
+        <li><code>spine-scripting</code>: <code id="spine-scripting"></code></li>
+      </ul>
+      <p>
+        If all entries are <code>true</code> or <code>false</code>, the test passes. 
```

Comment:
Nit: "false" is slightly unclear as a pass condition. I know this means "epubReadingSystem exists but the environment does not support, say, touch-events," but I think this would be clearer as something like enabled/disabled/unimplemented, with "unimplemented" as the test failure condition.

(Enabled/disabled doesn't seem like quite the right terminology either, but I'm blanking on the best word.)

Optional, though. As a minimal change, I would suggest just changing n/a to "unimplemented," because that's more explicit than n/a.

Reply:
Agree with 'unimplemented'. As for the other values: the [spec](https://www.w3.org/TR/epub-rs-33/#app-ers-hasFeature) defines these methods as returning a boolean value, ie, I think the best is to keep it as is.